include/nighthawk/adaptive_load/step_controller.h

```diff
@@ -1,3 +1,5 @@
+#pragma once
+
 // Interfaces for StepController plugins and plugin factories.
 
 #pragma once
```

Comment:
please remove.

Reply:
Done in 78338dca6de640193a4ea395ed2052c832397ea8.